pkg/operator/ceph/cluster/controller.go

```diff
@@ -70,6 +70,8 @@ const (
 	clusterDeleteRetryInterval = 2 // seconds
 	clusterDeleteMaxRetries    = 15
 	disableHotplugEnv          = "ROOK_DISABLE_DEVICE_HOTPLUG"
+	// disallowedHostDirectories directories which are not allowed to be used
+	disallowedHostDirectories = []string{"/etc/ceph", "/rook", "/var/log/ceph"}
```

Comment:
How come is this working? It shouldn't be possible to declare an array as a constant in Golang, unless I'm missing something...

Reply:
Fixed. My autocomplete and auto `go test` is broken since a few days, haven't gotten to get that fixed till now.